common/djangoapps/student/tests/test_parental_controls.py

```diff
@@ -60,13 +60,13 @@ def test_child_user(self):
         # Verify for a child born 13 years agp
         self.set_year_of_birth(current_year - 13)
         assert self.profile.requires_parental_consent()
-        assert self.profile.requires_parental_consent(date=datetime.date(current_year, 12, 31))
-        assert not self.profile.requires_parental_consent(date=datetime.date((current_year + 1), 1, 1))
+        assert self.profile.requires_parental_consent(year=datetime.date(current_year, 12, 31).year)
```

Comment:
Not `year=current_year`?

Reply:
... Yeah, I should have read that more closely. I meant to call out these changes too. Do you think there is concern in changing this method signature? Doesn't look like anything directly calls it in edx-platform, but I'm always scared with other repos.